src/mito2/src/test_util/sst_util.rs

```diff
@@ -132,7 +133,7 @@ pub fn new_batch_by_range(tags: &[&str], start: usize, end: usize) -> Batch {
         .unwrap()
 }
 
-pub fn new_batch_with_binary_by_range(tags: &[&str], start: usize, end: usize) -> Batch {
+pub fn new_batch_with_large_binary(tags: &[&str], start: usize, end: usize) -> Batch {
```

Comment:
Oh, I found that if we cast the array before writing the batch, we should use BinaryArray here.
```suggestion
pub fn new_batch_with_binary(tags: &[&str], start: usize, end: usize) -> Batch {
```